src/pages/index.js

```diff
@@ -37,61 +60,106 @@ profileEditButton.addEventListener("click", () => {
   editProfilePopup.open();
 });
 
+const editProfilePopup = new PopupWithForm(profileEditModal, (data) => {
+  editProfilePopup.saving(true);
+  api.updateUserInfo(data).then((data) => {
+    userInfo.setUserInfo(data.name, data.about);
+    editProfilePopup.saving(false);
+    editProfilePopup.close();
+    editFormValidator.toggleButtonState();
+  });
+});
+editProfilePopup.setEventListeners();
+
+addNewCardButton.addEventListener("click", () => {
+  addCardPopup.open();
+});
+
 const renderCard = (cardData) => {
   const cardElement = new Card(
```

Comment:
It's highly recommended to make a separate function `createCard` for creating a card not to copy-paste the code in the future when you want to add a card with `append` to follow  OOP principle `DRY` 

```js		

function createCard(item) {
    // here you create a card
  return cardElement.getView()
}

```

So, here will be


```js

const renderCard = (cardData) => {
  const cardElement = createCard(cardData);
  cardSection.addItem(cardElement);
};


```